app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```

Comment:
this file was use in chapter 6, so delete this file

Reply:
done